datetime/src/test/java/com/fasterxml/jackson/datatype/jsr310/ser/ZonedDateTimeSerTest.java

```diff
@@ -114,7 +114,14 @@ public void testSerializationAsTimestamp01NegativeSecondsWithDefaults() throws E
         ZonedDateTime original = ZonedDateTime.parse("Apr 13 1969 05:05:38.599 UTC", dtf);
```

Comment:
jsr310 is deprecated for jackson json - @pjfanning  

@cowtowncoder are there still updates to support this?

Reply:
Oops

Reply:
Yeah I think that despite deprecation, 2.x should make reasonable effort to keep it working still.